Learn-Information-Security-by-Building-a-Contact-Form/server090.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const app = express();
+const helmet = require("helmet");
+// https://helmetjs.github.io/ - for more information on all packages and uses.
+
+const helmetConfig = {
+  noCache: false
+};
+
+app.use(helmet(helmetConfig));
+// This will enable an additional feature disabling client-side caching.
```

Comment:
You could break this up into two short steps. The first could explain why disabling client-side caching is a good idea, then prompt learners to add the key `contentSecurityPolicy` with and empty object as its value to `helmetConfig`. The second step could be adding the `directives` object to `contentSecurityPolicy`.